packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -549,6 +554,16 @@ export function signInActor({ services }: SignInMachineOptions) {
         shouldSetupTOTP: (_, event): boolean => {
           return isExpectedChallengeName(getChallengeName(event), 'MFA_SETUP');
         },
+        shouldSelectMfaType: (_, event): boolean => {
+          console.error(
+            `ERROR: 'SELECT_MFA_TYPE' is not supported by Amplify UI yet.
+            Please use only one MFA type or set up the MFA selector using this document: https://docs.amplify.aws/lib/auth/mfa/q/platform/js/#advanced-use-cases/`
+          );
```

Comment:
> Should this be console.error or a just console.warn?

Should be `console.error` because it doesn't the expected result and requires the user to take actions.

> No need for a template literal here, can just be wrapped in double quotes

1. It's easier to read to cut the long line into multiple lines
2. don't have to add `\` or `\n`

> It's unclear to me what following the documentation linked here will provide the user in terms of not coming across this issue. Can you explain it to me?

I'm not sure I understand the question 😂